anglia-ruskin-erhvervsakademi-aarhus.csl

```diff
@@ -0,0 +1,710 @@
+          <group>
+            <text variable="volume" prefix=" "/>
+            <text variable="authority" prefix=" (" suffix=")"/>
+          </group>
+          <group>
+            <label variable="page" form="short"/>
+            <text variable="page"/>
+          </group>
+        </if>
+      </choose>
+    </group>
+  </macro>
+  <macro name="locator">
+    <choose>
+      <if type="article-journal article-newspaper article-magazine interview" match="any">
+        <group delimiter=", " suffix=".">
+          <group>
+            <text variable="volume"/>
+            <text variable="issue" prefix="(" suffix=")"/>
+          </group>
+          <choose>
+            <if type="article-magazine article-newspaper interview" match="any">
+              <date variable="issued" delimiter=" ">
+                <date-part name="day"/>
+                <date-part name="month" form="short"/>
+              </date>
+            </if>
+          </choose>
+          <choose>
+            <if variable="page">
+              <group>
+                <label variable="page" form="short"/>
+                <text variable="page"/>
+              </group>
+            </if>
+          </choose>
+        </group>
+      </if>
+      <else-if type="book chapter paper-conference manuscript entry-dictionary entry-encyclopedia thesis motion_picture song report" match="any">
+        <group suffix=".">
+          <text variable="event" suffix=". "/>
+          <group delimiter=".">
+            <text macro="publisher"/>
+            <group>
+              <label variable="page" form="short"/>
+              <text variable="page"/>
+            </group>
+          </group>
+        </group>
+      </else-if>
+      <else-if type="broadcast">
+        <group delimiter=". ">
+          <text variable="event"/>
+          <text macro="publisher"/>
+          <group>
+            <date variable="issued" delimiter=" ">
+              <date-part name="day"/>
+              <date-part name="month" form="short"/>
+            </date>
+          </group>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="online-access">
+    <choose>
+      <if variable="URL">
+        <group>
+          <text term="available at" text-case="capitalize-first" prefix=" " suffix=": "/>
+          <text variable="URL" prefix="&lt;" suffix="&gt;"/>
+          <group prefix=" [" suffix="].">
+            <text term="accessed" text-case="capitalize-first" suffix=" "/>
+            <date variable="accessed" delimiter=" ">
+              <date-part name="day"/>
+              <date-part name="month" form="short"/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="author-short">
+    <choose>
+      <if type="bill legal_case legislation" match="any">
+        <text variable="title"/>
+      </if>
+      <else-if match="none" variable="author">
```

Comment:
That's probably not right as it will mark edited volumes as Anon ( see the Hancke et al. in the preview)